html/themes/custom/common_design_subtheme/components/rw-document/rw-document.css

```
@@ -129,22 +129,49 @@
  * a button like it was on the RW7 site. */
 .rw-document.rw-article--book .rw-article__content a[href^="/user/login/"] {
   display: inline-block;
-  margin: 1rem 0 0 0;
+  margin: 1rem 0 0 1rem;
   padding: 12px 18px;
   color: white;
   border: none;
-  border-radius: 3px;
+  border-radius: 2rem;
   background: var(--cd-reliefweb-brand-blue--dark);
   font-weight: bold;
   text-decoration: none;
 }
+
+.rw-document.rw-article--book .rw-article__content a[href^="/user/login/"]::before {
+  position: relative;
+  top: 1px;
```

```suggestion
  top: 2px;
```

Hmmm. On closer inspection the ::before top should probably be 2px and not 1px so as to not offend my eyes :-)